### DISCUSSION TITLE
Suggestion: using light background for code blocks

### DISCUSSION THREAD

**Discussion by hzhsbyy** [rails/rails#52857] [Docs]
@AmandaPerino 
I really like the new style of Rails guides, but one thing bothers me is that all the code blocks are in black background.

The high contrast between dark code blocks and the remaining light parts makes my eyes sick. One of the symptoms is the [persistence of vision](https://en.wikipedia.org/wiki/Persistence_of_vision).

I wish you can change the background code of code blocks back as the previous version did.

Also, I found a discussion topic on [dark mode-themed code blocks on dark mode](https://github.com/rails/rails/discussions/51378). So I think my request is reasonable.

Thanks!

**Comment by hzhsbyy** [comment]
I currently read markdown files from GitHub directly (e.g. https://github.com/rails/rails/blob/main/guides/source/engines.md), which also has the advantage of the main content being centered.

I think the `Chapters` part should really be placed on the left instead of on the right. Or, there could be a index of all guides on the left so that the main content is centered for the ease of reading.

**Comment by JuanVqz** [comment]
Is this still a valid suggestion?

**Reply by Wheres-my-mind** [reply]
yep
